paket.dependencies

```diff
@@ -1,24 +1,36 @@
 redirects: on
+storage: none
 source https://www.nuget.org/api/v2/
 
-// We want to depend on FSharp.Core 4.0.0.1, but not require
-// that version in the generated nuget package (see paket.template).
+// We want to depend on FSharp.Core 4.1.17 (it's the earliest one
+// that supports .NET standard), but not require that version in
+// the generated nuget package (see paket.template).
 //
-// If we just specify "FSharp.Core 4.0.0.1" then the nuget package
+// If we just specify "FSharp.Core 4.1.17" then the nuget package
 // has an dependency on that *exact* version.
 //
-// So, we specify 4.0.0.1 as a minimum bound, and set lowest_matching
+// So, we specify 4.1.17 as a minimum bound, and set lowest_matching
 // so it won't be updated automatically to a newer one by 'paket update'.
-nuget FSharp.Core >= 4.0.0.1 lowest_matching: true
+nuget FSharp.Core >= 4.1.17 lowest_matching: true
```

Comment:
This means we're going to target F# 4.1?

Reply:
This is the first `FSharp.Core` version that supports .NET Standard.

Reply:
Oh, I see.